index.html

```diff
@@ -40,6 +40,181 @@ <h1>Hey there. I'm Sainingo <br><span>I'm a software developer</span></h1>
+            </div>
+        </div>
+
+        <!-- fifth container -->
+        <div class="container">
+            <div class="card">
+               <img src="./assets/images/placeholder.png" alt="image placeholder" >
+               <div class="desc">
+                   <h4>Muilti-Post Stories Gain+Glory</h4>
+                   <ul>
+                       <li>Ruby on rails</li>
+                       <li>css</li>
+                       <li>Javascript</li>
+                       <li>html</li>
+                   </ul>
+               </div>
+               <button type="button">See Project</button>
+            </div>
+        </div>
+
+        <!-- sixth container -->
+        <div class="container">
+            <div class="card">
+               <img src="./assets/images/placeholder.png" alt="image placeholder" >
+               <div class="desc">
+                   <h4>Muilti-Post Stories Gain+Glory</h4>
+                   <ul>
+                       <li>Ruby on rails</li>
+                       <li>css</li>
+                       <li>Javascript</li>
+                       <li>html</li>
+                   </ul>
+               </div>
+               <button type="button">See Project</button>
+            </div>
+        </div>
+     </section>
+
+     <!-- About me section -->
+     <section>
+         <div class="upper-bg"></div>
+         <div class="about-me">
+            <h4>About Me</h4>
+            <p>I can help you build a product, feature or website look through some of my work and experience!
+                if you like what you see and have a project you need coded, dont hesitate to contact me.
+            </p>
+            <button type="button">Get My Resume</button>
+         </div>
+         <!-- cards -->
+         <div class="card-section">
+             <img src="./assets/images/icon-lang.png" alt="image placeholder">
+             <p>Languages</p>
+             <ul>
+                 <li>Javascript</li>
+                 <li>Ruby</li>
+                 <li>Html</li>
+                 <li>css</li>
+             </ul>
+         </div>
+         <!-- card-1 -->
+         <div class="card-section1">
+            <img src="./assets/images/icon-frame.png" alt="image placeholder">
+            <p>Frameworks</p>
+            <ul class="card-1">
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li>Selenium</li>
+            </ul>
+        </div>
+        <!-- card-2 -->
+        <div class="card-section2">
+            <img src="./assets/images/icon-skills.png" alt="image placeholder">
+            <p>Skills</p>
+            <ul class="card-2">
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+        </div>
+     </section>
```

Comment:
- [ ] There seems to be a vertical scroll on the page. Kindly remove or prevent the scroll as it affects the user experience.

![image](https://user-images.githubusercontent.com/43995830/154137556-377c96b9-e1fb-4915-bc88-6c1bb3134f75.png)

Reply:
Thanks was able to fix

Reply:
👍🏽 Good now.